dist/index.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Hello World</title>
-</head>
+<link href="main.css" rel="stylesheet"></head>
```

Comment:
Why is this here and not in an import?

Reply:
This file is automatically generated by webpack configuration.

Reply:
Ok